source/npmhub.js

```diff
@@ -56,6 +57,13 @@ async function init() {
       '.repository-content', // Old container https://github.com/refined-github/refined-github/issues/5751
     ]),
   });
+
+  const buttonTarget = document.querySelector(['.BorderGrid-cell [href=\'#readme\']']).parentNode;
```

Comment:
The sidebar doesn't exist on the page:

- https://github.com/npmhub/npmhub/blob/main/package.json

So this will cause an error. I suggest changing this to:

```suggestion
  const position = document.querySelector(['.BorderGrid-cell [href=\'#readme\']'])?.parentElement;
```

And then wrap the following code with `if (position) {`